### PR TITLE
[back] feat: create the `faq` app with its models

### PR DESCRIPTION
**Pull request by GresilleSiffle**
**related to** #1068 

---

This PR adds the `faq` app, question/answer models and the related administration interface.

In the admin interface, questions can be ordered by rank, and an icon displays if the question has an answer or not yet. Both questions and answers can be translated.

### preview

capture

capture2

### CODE REVIEW

File: backend/faq/models.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    )
+    question = models.OneToOneField(
+        FAQuestion, on_delete=models.CASCADE, related_name="answer"
+    )
+
+    class Meta:
+        verbose_name = "FAQ Answer"
+
+    def __str__(self) -> str:
+        return self.name
```
[the remark this answers is not in the document]
Comment (lfaucon) [reply backend/faq/models.py:63]: Sounds good to me

[the remark this answers is not in the document]
Comment (GresilleSiffle) [reply backend/faq/models.py:63]: Good point. I think I had in mind the possibility to have many answers but as we decided to have only one we can indeed remove `FAQAnswer`.

**Review by lfaucon — APPROVED** [review]
LGTM

File: backend/faq/models.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (GresilleSiffle) [reply backend/faq/models.py]: Fixed by https://github.com/tournesol-app/tournesol/pull/1105/commits/06bfee7c026cb0655297921ac72644aa2b858ef9.

**Pull request merged**